src/app/detail/ui/Sidebar.tsx

```diff
@@ -79,7 +81,15 @@ export const Sidebar = ({
     }
   }, [tasks, workflowStates])
 
-  const matches = useMediaQuery('(max-width:600px)')
+  const windowWidth = useWindowWidth()
+  const isMobile = windowWidth < 600 && windowWidth !== 0
+
+  useEffect(() => {
+    if (isMobile) {
+      store.dispatch(setShowSidebar(false))
+    }
+  }, [isMobile])
```

Comment:
We seem to have sm breakpoint for this. I think using that would give you much better result.